versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     equinix = {
       source  = "equinix/equinix"
-      version = "~> 1.14.0"
+      version = "1.20.0"
```

Comment:
It's OK to lock down the provider version in `example` modules, but this version constraint shouldn't change; if we don't allow flexibility here then we have to cut a new release of this module every time we cut a new terraform provider release.

Reply:
Thanks! Changed it to `"~> 1.20.0"` in both root and examples modules in the next commit 692fcccb695b63eab1af72e15a3d8017484e9cf9